src/libcore/ops/function.rs

```diff
@@ -127,27 +145,35 @@ pub trait FnMut<Args> : FnOnce<Args> {
     extern "rust-call" fn call_mut(&mut self, args: Args) -> Self::Output;
 }
 
-/// A version of the call operator that takes a by-value receiver.
+/// The version of the call operator that takes a by-value receiver.
+///
+/// Instances `FnOnce` can be called, but might not be callable multiple times.
```

Comment:
Reading over this again, i think it'll work. However, "Instances \[of\] `FnOnce`", missing an "of".

r=me once that's done!